sdk/program/src/system_instruction.rs

```diff
@@ -307,6 +307,13 @@ pub enum SystemInstruction {
         /// Owner to use to derive the funding account address
         from_owner: Pubkey,
     },
+
+    /// One-time idempotent upgrade of legacy nonce versions in order to bump
```

Comment:
what does this `idempotent` refers to? i mean, this instruction impl. errors `InstructionError::InvalidArgument` if `Current` nonce account (i.e. already upgraded) is given. So, you cannot invoke `UpgradeNonceAccount` multiple times successfully, given the same nonce account.